frontend/src/pages/EditRules/index.tsx

```diff
@@ -45,7 +49,20 @@ function EditRules(): JSX.Element {
 		ruleId == null ||
 		(data?.payload?.data === undefined && !isLoading)
 	) {
-		return <div>{data?.error || t('something_went_wrong')}</div>;
+		return (
```

Comment:
@rahulkeswani101 : We are not sure that in all cases the alert would be deleted. Check for error and then show corresponding error message

Reply:
+1

Reply:
https://signoz-team.slack.com/archives/C02TJ466H8U/p1721820237659469?thread_ts=1721819096.698889&cid=C02TJ466H8U @rahulkeswani101 please follow this!